pages/events/ownedEvents.js

```diff
@@ -0,0 +1,21 @@
+import { useSession } from "next-auth/react";
+import EventList from "@/components/EventList";
+import useSWR from "swr";
+import Loading from "@/components/Loading";
+import Error from "@/components/Error";
+import Button from "@/components/Button";
+import { useRouter } from "next/router";
+export default function MyEvents() {
+  const { data: events, isLoading, error } = useSWR("/api/users/me/events");
+  const router = useRouter();
+  if (isLoading) return <Loading />;
+  if (error) return <Error>{error.message}</Error>;
+
+  return (
+    <>
+      <h1>My Events</h1>
```

Comment:
pay attention to semantic meanings. Only one h1 per application. The most important one, since its relevant for SEO and a11y.